### PR TITLE
Added additonal information for installation and usage to make setup …

### PR DESCRIPTION
**Pull request by mrt181**
…easier. #346

### CODE REVIEW

**Comment by TamalSaha** [comment]
Since 1.8, GOPATH env var is not required. https://github.com/golang/go/issues/17262 . A backward compatible alternative is `$(go env GOPATH)`

File: README.md [hunk 1]
```diff
@@ -35,6 +36,8 @@ make
 make check
 sudo make install
 ```
+If you are using cygwin and ProtocolBuffers 3.0.0-beta-3 or later is not available for cygwin yet install the Windows version.
```
Comment (tmc) [inline README.md]: would it be appropriate to place windows instructions in its own section?

File: README.md [hunk 2]
```diff
@@ -128,7 +136,7 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
    
    It will generate a reverse proxy `path/to/your_service.pb.gw.go`.
 
-   Note: After generating the code for each of the stubs, in order to build the code, you will want to run ```go get .``` from the directory containing the stubs.
+   Note: After generating the code for each of the stubs, in order to build the code, you will want to move the stubs into your `$GOPATH/src` working directory and run ```go get .``` from the directory containing the stubs.
```
Comment (tmc) [inline README.md]: how about we recommend that folks work inside of GOPATH over moving files after gen?

File: README.md [hunk 3]
```diff
@@ -186,6 +194,12 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
      --swagger_out=logtostderr=true:. \
      path/to/your_service.proto
    ```
+8. Generate executable
+
+   Run this inside the directory containg your `main.go`.
+   ```
+   go build
```
Comment (tmc) [inline README.md]: let's recommend go install instead

File: README.md (hunk 2 above)
Comment (mrt181) [inline README.md]: the protoc command creates the stubs in the same directory as the service.proto file. is it possible to provide an output directory for the protoc plugins?

**Comment by tmc** [comment]
@mrt181 still need a CLA sign here

**Comment by mrt181** [comment]
Already done?
Agreement 	Name 	Date Signed
Google Individual CLA 	Martin Thomalla 	Apr 04, 2017 14:28 PDT

**Comment by tmc** [comment]
@mrt181 thanks! there are a few outstanding comments/suggestions.

**Comment by mrt181** [comment]
I would suggest to reject this PR. 
While redoing the installation on anothe machine I experienced a more involved installation process for cygwin. I have created a wiki page for cygwin installation documenting the different steps that were required to `go install` an grpc-gateway service.

**Comment by tmc** [comment]
@mrt181 thanks for the great documentation contribution!

**Pull request closed without merging**